examples/factory-contract/Cargo.toml

```diff
@@ -6,16 +6,20 @@ edition = "2021"
 
 [dev-dependencies]
 anyhow = "1.0"
-near-sdk = { path = "../../near-sdk" }
+near-sdk = { version = "4.1.1", features = ["unstable"] }
```

Comment:
What `unstable` feature is used in this example? Examples should avoid using unstable features unless they specifically want to feature that specific feature.